Source/Targeting.swift

```diff
@@ -181,6 +181,40 @@ import CoreLocation
         }
     }
     
+    // MARK: - External UserIds
+    var externalUserIds = [ExternalUserId]()
+    /**
+     * This method allows to save External User Id in the User Defaults
+     */
+    public func setExternalUserId(_ externalUserId: ExternalUserId) {
```

Comment:
@avermaiOS @ppuviarasu 
Shouldn't we allow over-writing the external user id for a given source?

Reply:
Yes, we should. Made the changes accordingly